product_docs/docs/pem/8/pem_schema_diff_tool.mdx

```diff
@@ -21,65 +21,69 @@ The Schema Diff feature allows you to:
 !!! Note
     The source and target databases must be of the same major version.
 
-Click on `Schema Diff` under the `Tools` menu to open a selection panel. Choose the source and target servers, databases, and schemas that will be compared. After selecting the objects, click on the `Compare` button.
+Select **Tools > Schema Diff** and a selection panel opens. Choose the source and target servers, databases, and schemas to compare. After choosing the objects, select **Compare**.
 
-You can open multiple copies of `Schema Diff` in individual tabs simultaneously. To close a tab, click the `X` in the upper-right hand corner of the tab bar.
+You can open multiple copies of Schema Diff in individual tabs simultaneously. To close a tab, select **X** in the upper-right corner of the tab bar.
 
 ![Schema Diff dialog](images/schema_diff_dialog.png)
 
-Use the `Preferences` dialog to specify if `Schema Diff` should open in a new browser tab. Set `Open in new browser tab` option to `true`.
+To specify whether to open Schema Diff in a new browser tab, select the Preferences dialog box. Set **Open in new browser tab** option to **true**.
 
-The `Schema Diff` panel is divided into two panels; an `Object Comparison panel` and a `DDL Comparison panel`.
+The Schema Diff panel is divided into two panels: an Object Comparison panel and a DDL Comparison panel.
 
-## Schema Diff object comparison panel
+## Schema Diff Object Comparison panel
 
-In the object comparison panel, you can select the source and target servers of the same major version, databases, and schemas to be compared. You can select any server listed under the browser tree whether it is connected or disconnected. If you select a server that is not connected then it will prompt you for the password before using the server.
+In the Object Comparison panel, you can select the source and target servers of the same major version, databases, and schemas to compare. You can select any server listed under the browser tree whether it is connected or disconnected. If you select a server that isn't connected, then you must enter the password before using that server.
 
-Next, select the databases that will be compared. The databases can be the same or different (and within the same server or from different servers).
+Next, select the databases to compare. The databases can be the same or different (and from the same server or different servers).
```

Comment:
It may be complicated to make this into a procedure because there are steps farther down about filtering and viewing the comparison results. We'd have to decide whether to include them in the procedure or as a separate one.